### PR TITLE
Features

### PR DESCRIPTION
**Pull request by renercrows**
### Desktop-Portfolio

- Added media queries.
- Added some HTML div tags for background images.
- Added CSS to make its desktop version.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+            <li class="column">
+                <a href=""><img src="images/medium.png" alt="medium"></a>
+            </li>
+        </ul>
+    </section>
+    <section>
+        <div class="spacing">
+            <a class="disabled" href=""><img src="images/Disabled.png" alt="next"></a>
+        </div>
+        <h2 class="subtitle2">My Recent Works</h2>
```
Comment (Kossi-stack) [inline index.html:46]: - [ ] Kindly fix the work section
- Your Design
Capturedest
- Figma Design
Capturefdes

File: index.html [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+                        </ul>
+                        <button type="submit" class="button1">See Project</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="middle2">
+        <div class="background2">
+                <h2 class="subtitle3">About me</h2>
```
Comment (Kossi-stack) [inline index.html:141]: - [ ] Kindly fix the about me section to look the same as the Figma design
- Your Design
Capturedesto

- Figma Design
Capturefide

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @rachidelaid, 

Good job so far!

- All linters are passing
- You are following GitHub flow
- Good commit history
- Good PR description
- Your Design is looking good

There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️

- [ ]  Please fix the mobile version requested by the previous reviewer.
your design
fix_screen




_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @renercrows , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Kindly please fix the mobile version requested by the previous reviewer.
your design
Screenshot from 2022-06-03 20-37-43



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by renercrows** [comment]
@ShahierNashaat 

Changes were made.

**Review by sinansevgi — COMMENTED** [review]
Hello @renercrows  👋,

_This is an additional review as a result of the second opinion request._

I can see that the mobile version works as expected. :+1: 

As this was the only reason that your project was not approved it will be **approved** as soon as possible.
Please note that since we will approve it **manually** so this could **take a few hours**!

Keep rocking 🚀!

**Pull request merged**